src/I_Ecobee1.xml

```diff
@@ -954,7 +954,7 @@ decompress_lzo_file() {
       if isEmsThermostat(lul_device) then
         return away and setOccupied(session, selection, lul_device, false) or resumeProgram(session, selection, 3)
       else
-        return setClimateHold(session, selection, lul_device, away and "away" or "home")
+        return setClimateHold(session, selection, lul_device, away and "away" or "home" or "sleep" or "smart1")
```

Comment:
The old line 957 means: if away is not false, set a climate hold to "away", else set a climate hold to "home".  What do you want to do here regarding "sleep" or "smart1"?